### ISSUE TITLE
Can we export the whole call stack to a text file

### ISSUE THREAD

**Issue by kannanekanath** [Vanco/SequencePlugin#34]
Often when you work in a large legacy project and especially if you are re-writing them you want to explain what an entire piece of code does. I generated a sequence diagram and as I guessed it simply created a massive diagram that is going to be difficult for my team to comprehend. 

What would be great is if the call stack can be exported to a hierarchical/tabbed text file and then with some filtering I can create a simplified flow text. The team will migrate different portions of the code in different timelines but having a textual reference will be useful.

In short can we somehow plugin a text based generator instead of the diagram?

**Comment by kannanekanath** [comment]
Thank you for the fast turnaround I never expected this. It looks great. I only suggest a very minor thing which is to simply use spaces instead of the tab charactar

**Comment by UDAYADAV** [comment]
@Vanco @kannanekanath I don't see a way to import the generated text file. Could you please help?

**Comment by Vanco** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        There is no import function yet.   the exported text only for somebody who need read detail, it is not exchange format with other uml tools.
        
    
        
            
        
    
    
     
    
        
                    
            
                
                    
                            
                                
                            
                            
                                fanhuagang
                            
                    
                        
                            
                                    fanhuagang@gmail.com
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 

    On 3/18/2020 13:49，Udaya Kumar Yadav<notifications@github.com> wrote： 



@Vanco @kannanekanath I don't see a way to import the generated text file. Could you please help?

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
